inference-engine/src/legacy_api/src/ie_layer_validators.cpp

```diff
@@ -1186,7 +1186,7 @@ void NMSValidator::parseParams(CNNLayer* layer) {
 
 #define REG_LAYER_VALIDATOR_FOR_TYPE(__validator, __type) _validators[#__type] = std::make_shared<__validator>(#__type)
 
-LayerValidators::LayerValidators() {
+LayerValidators::LayerValidators() : _validators() {
```

Comment:
Why do we need this line? I thought that for map in any case default constructor is used, am I not right?